### ISSUE TITLE
Validate tree-ware to protobuf mappings

### ISSUE THREAD

**Issue by deepak-nulu** [tree-ware/tree-ware-kotlin-proto3#3]
The protobuf paths specified in a mapping need to be validated to make sure that they exist in the `.proto` files. Implement a `validate()` function that is passed the meta-model with the mapping, and the file paths to the `.proto` files specified in the mapping. The function should log errors with the meta-model element's `fullName` and the erroneous absolute proto path.

The protobuf mapping will have to be used in an operator to convert tree-ware models into protobufs (#4). It will be hard for operators to operate on a model and a meta-model since they are different shapes. So the `validate()` function should return an archetype model with mapping info in the archetype model's aux data. The operator will then be able to use this archetype model to perform the conversion.

The `validate()` function needs to be called only once when the system starts up (since the mapping is pre-defined and not changed at runtime).

Blocked by #2

**Comment by deepak-nulu** [comment]
Entity and field mappings are done. Enum mappings are not done. Will open another ticket for it. Marking this as closed.
